modules/data.remote/tests/Rcheck_reference.log

```diff
@@ -33,22 +80,47 @@
 * checking whether the namespace can be loaded with stated dependencies ... OK
 * checking whether the namespace can be unloaded cleanly ... OK
 * checking loading without being on the library search path ... OK
+* checking use of S3 registration ... OK
 * checking dependencies in R code ... WARNING
 'library' or 'require' calls in package code:
   ‘PEcAn.DB’ ‘doParallel’ ‘raster’ ‘rgdal’
   Please use :: or requireNamespace() instead.
   See section 'Suggested packages' in the 'Writing R Extensions' manual.
-Missing or unexported objects:
-  ‘PEcAn.utils::download_file’ ‘PEcAn.utils::need_packages’
+Namespace in Imports field not imported from: ‘stringr’
+  All declared Imports should be used.
 * checking S3 generic/method consistency ... OK
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
+call_MODIS: no visible global function definition for ‘write.csv’
 download.LandTrendr.AGB: no visible binding for global variable ‘k’
+download.NLCD: no visible global function definition for ‘dbfile.check’
+download.NLCD: no visible global function definition for ‘db.query’
+download.NLCD: no visible global function definition for
+  ‘dbfile.insert’
+download.thredds.AGB : get_data: no visible global function definition
+  for ‘write.csv’
+download.thredds.AGB: no visible global function definition for
+  ‘foreach’
+download.thredds.AGB: no visible global function definition for
+  ‘stopCluster’
+extract_NLCD: no visible global function definition for ‘dbfile.check’
+extract_NLCD: no visible global function definition for ‘db.query’
+extract_NLCD: no visible global function definition for ‘raster’
+extract_NLCD: no visible global function definition for ‘SpatialPoints’
+extract_NLCD: no visible global function definition for ‘CRS’
+extract_NLCD: no visible global function definition for ‘spTransform’
+extract_NLCD: no visible global function definition for ‘crs’
+extract_NLCD: no visible global function definition for ‘extract’
```

Comment:
Why are a bunch of new warnings being added to the log? I thought the ideas was that we would only ever reduce them.

Reply:
These were accidentally added way up at the top of the PR -- notice this is the log for data.remote in a data.atmosphere PR. Sure thought we'd removed them before, but will do it now.

Reply:
Fixed in [f6b0857](https://github.com/PecanProject/pecan/pull/3218/commits/f6b08570fe7654f0e6c2c61ed09c8ae5620352cb)